### ISSUE TITLE
OpenSearch Dashboards - pdf not downloading behind a gateway

### ISSUE THREAD

**Issue by ARud12** [opensearch-project/OpenSearch-Dashboards#1636]
I have a gateway, where I am using an internal and external URL (differs from each other). I am assuming that OpenSearch Dashboards are checking if the URL's exist internally as well as externally. In my case the internal URL will exist internally but will fail if accessed externally, the same with the external URL. This is causing that the pdf's cannot be downloaded behind the gateway. Any advice?

**Comment by kavilla** [comment]
Hello @ARud12,

Thanks for opening! I believe you are talking about the reporting plugin is not able to download a pdf behind a gateway?

Also, could you provide more insight on which version you are running, more information on environment that is running OpenSearch Dashboards, browser and OS.

Thanks!

**Comment by ARud12** [comment]
Yes, running on Windows, version 1.3.2. Mostly testing on Chrome.

**Comment by ARud12** [comment]
OpenSearch dashboards is hosted internally on a different URL than it is accessed by externally. And that is causing errors because it is seems as if OpenSearch dashboards checks if the URL exists internally and externally and in my case it is not the same. The error that I am getting is ERR_NAME_NOT_RESOLVED

**Comment by ARud12** [comment]
Is there any feedback on this issue?

**Comment by kavilla** [comment]
Hello @ARud12,

Apologies on the delay. I will re-route to the reporting plugin. Could you provide steps to recreate the bug?

Thank you.
